code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -235,6 +239,36 @@
 			new /obj/item/gun/energy/wormhole_projector/core_inserted(src) // 5 tc easily
 			new /obj/item/gun/energy/decloner/unrestricted(src) // 5 tc at least also
 
+		if(KIT_MAID)//ORBSTATION CHANGE,
+			new /obj/item/clothing/under/syndicatemaid(src)
+			new /obj/item/clothing/gloves/syndiemaid(src)
+			new /obj/item/clothing/accessory/syndiecorset(src)
+			new /obj/item/clothing/shoes/chameleon/noslip(src)
+			new /obj/item/grenade/clusterbuster/soap(src)
+			new /obj/item/grenade/clusterbuster/soap(src)
+			new /obj/item/grenade/chem_grenade/ez_clean(src)
+			new /obj/item/grenade/chem_grenade/ez_clean(src)
+			new /obj/item/grenade/chem_grenade/ez_clean(src)
+			new /obj/item/soap/syndie(src)
+			new /obj/item/storage/belt/chameleon(src)
+			new /obj/item/reagent_containers/spray/syndicate/lube(src)
+			new /obj/item/pushbroom/syndicate(src)
+
+		if(KIT_PYRO)//ORBSTATION CHANGE,
+			new /obj/item/flamethrower/full/tank(src)
+			new /obj/item/tank/internals/plasma(src)
+			new /obj/item/clothing/mask/gas/atmos(src)
+			new /obj/item/clothing/suit/armor/gorlexcoat(src)
+			new /obj/item/clothing/under/syndicateoveralls/fireproof(src)
+			new /obj/item/clothing/gloves/combat(src)
+			new /obj/item/clothing/shoes/combat(src)
+			new /obj/item/gun/ballistic/automatic/pistol(src)
+			new /obj/item/ammo_box/magazine/m9mm/fire(src)
+			new /obj/item/ammo_box/magazine/m9mm/fire(src)
+			new	/obj/item/grenade/chem_grenade/incendiary(src)
+			new	/obj/item/grenade/chem_grenade/incendiary(src)
+			new	/obj/item/grenade/chem_grenade/incendiary(src)
+
```

Comment:
is this indentation stuff just git being weird? thats the only worry i have got

Reply:
yes it is